src/utils/hooks/api.js

```diff
@@ -29,7 +29,7 @@ export const useGetReferentiel = (nomenclatures) => {
   const getReferentielForVizu = useConstCallback((refName) => {
     if (nomenclatures && Object.keys(nomenclatures).includes(refName)) {
       const finalUrl = nomenclatures[refName];
-      return getFetcherForLunatic(oidc.getTokens().accessToken)(finalUrl);
+      return getFetcherForLunatic(null)(finalUrl);
     }
```

Comment:
I believe this fetcher is utilized in a collected mode that requires a token to obtain nomenclature from the API. In the unauthenticated mode, the dummy client should return null as indicated [here](https://github.com/InseeFr/Stromae/blob/8803fede3973218d97c0d786f84a426e3b56e242/src/components/auth/provider/component.js#L11). We need to investigate the reason for the error being thrown.

Reply:
This function is used only in vizualisation mode, not in collected mode.
The collected mode always use the function : `getReferentiel` (not `getReferentielForVizu`)

Reply:
ok good